docs/conch/examples/sshsimpleserver.py

```diff
@@ -46,6 +31,22 @@
 $ ssh -p 5022 -i ssh-keys/client_rsa user@localhost
 """
 
+import sys
+
+from zope.interface import implementer
+
+from twisted.conch import avatar
+from twisted.conch.checkers import InMemorySSHKeyDB, SSHPublicKeyChecker
+from twisted.conch.ssh import connection, factory, keys, session, userauth
+from twisted.conch.ssh.transport import SSHServerTransport
+from twisted.cred import portal
+from twisted.cred.checkers import InMemoryUsernamePasswordDatabaseDontUse
+from twisted.internet import protocol, reactor
+from twisted.python import components, log
+
+log.startLogging(sys.stderr)
+
+
```

Comment:
Also wrong, these are examples written in this order for a reason.

Reply:
I'll just revert all the docstring stuff &mdash; the `check-docstring-first` check has tons of false positives.

Reply:
> I'll just revert all the docstring stuff — the `check-docstring-first` check has tons of false positives.

Ah, yeah, let's get rid of that.